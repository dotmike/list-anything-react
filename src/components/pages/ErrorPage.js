import React from "react";

import CenterElemsContainer from "../containers/CenterElemsContainer";
import ErrorMessage from "../messages/ErrorMessage";

const ErrorPage = () => (
  <CenterElemsContainer>
    <ErrorMessage text={"Please, try again later..."} />
  </CenterElemsContainer>
);

export default ErrorPage;
