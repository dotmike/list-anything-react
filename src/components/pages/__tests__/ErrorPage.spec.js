import React from "react";
import { shallow } from "enzyme";

import ErrorPage from "../ErrorPage";

describe("ErrorPage", () => {
  it("renders correctly when `error` is passed", () => {
    const wrapper = shallow(<ErrorPage error="some error" />);
    expect(wrapper).toMatchSnapshot();
  });

  it("renders correctly when `error` is not passed", () => {
    const wrapper = shallow(<ErrorPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
