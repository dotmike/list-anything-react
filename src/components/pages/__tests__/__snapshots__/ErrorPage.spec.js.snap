// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage renders correctly when \`error\` is not passed 1`] = `
<CenterElemsContainer>
  <ErrorMessage
    showHeader={true}
    text="Please, try again later..."
  />
</CenterElemsContainer>
`;

exports[`ErrorPage renders correctly when \`error\` is passed 1`] = `
<CenterElemsContainer>
  <ErrorMessage
    showHeader={true}
    text="some error"
  />
</CenterElemsContainer>
`;
